app/models/protocol.rb

```diff
@@ -15,6 +17,11 @@ class Protocol < ApplicationRecord
     in_repository_archived: 4
   }
 
+  scope :recent_protocols, lambda { |user, team, amount|
+    where(team: team, protocol_type: 3).or(where(team: team, protocol_type: 2, added_by: user))\
```

Comment:
You can use :in_repository_private instead of simple number and there is no need to put backslash in the end of the line

Reply:
fix